### PR TITLE
feat(webhooks.md): documentation for a new event

### PR DESCRIPTION
**Pull request by afortune4**
](https://iadvize.atlassian.net/browse/INT-3121)

### CODE REVIEW

File: docs/webhooks.md [hunk 1]
```diff
@@ -158,6 +159,25 @@ Emitted when user is disconnecting to administration or desk.
 }
 </pre>
 
+
+### `user.availability.updated`
+
+Emitted when user's availability is updated from desk.
```
Comment (sdemonti) [inline docs/webhooks.md:165]: Not only from desk.

```suggestion
Emitted when user's availability is updated.
```

File: docs/webhooks.md [hunk 2]
```diff
@@ -158,6 +159,25 @@ Emitted when user is disconnecting to administration or desk.
...
+
+<pre class="prettyprint lang-js">{
+    "eventId": "d36cd3c4-2d16-4a77-97c2-620bde859b29",
+    "eventType": "user.availability.updated",
+    "platform": "sd",
+    "projectId": 3030,
```
Comment (sdemonti) [inline docs/webhooks.md:171]: I think @alexduf was right, if we can, we must avoid to indicate real projectId.

```suggestion
    "projectId": 1,
```

File: docs/webhooks.md [hunk 3]
```diff
@@ -158,6 +159,25 @@ Emitted when user is disconnecting to administration or desk.
...
+    "clientId": 1,
+    "createdAt": "2017-04-22T11:01:00+02:00",
+    "sentAt": "2017-04-22T11:01:00+02:00",
+    "operatorId": 15253,
```
Comment (sdemonti) [inline docs/webhooks.md:175]:
```suggestion
    "operatorId": 1,
```

File: docs/webhooks.md [hunk 4]
```diff
@@ -158,6 +159,25 @@ Emitted when user is disconnecting to administration or desk.
...
+    "availabilities": [{"communicationMode": "CHAT", "status": "AVAILABLE"}]
```
Comment (sdemonti) [inline docs/webhooks.md:176]: When we have enumeration, we add a note with all enumeration possible below the payload.

For example, i could be:

```markdown
Please note :
| Attribute | Description |
| --- | --- |
| communicationMode | Could be CHAT, CALL, VIDEO or THIRD_PARTY |
| status | Could be AVAILABLE or UNAVAILABLE|
```

**Review by sdemonti — CHANGES_REQUESTED** [review]

File: docs/webhooks.md [hunk 5]
```diff
@@ -158,6 +159,25 @@ Emitted when user is disconnecting to administration or desk.
...
+
```
Comment (sdemonti) [inline docs/webhooks.md:177]: Missing `origin` field too !

**Review by sdemonti — CHANGES_REQUESTED** [review]

File: docs/webhooks.md (hunk 5 above)
Comment (maylencita) [reply docs/webhooks.md:177]: Actually the `origin` field is missing from the event ^^'
I commented on the accompanying PR

**Review by maylencita — APPROVED** [review]

File: docs/webhooks.md [hunk 6]
```diff
@@ -50,10 +51,10 @@ Emitted on a beginning of a conversation or a receiving of a conversation transf
   "eventId": "0f0bb3af-5035-4ba3-b3fb-ff4879a3a74d",
   "eventType": "v2.conversation.pushed",
   "platform": "ha",
-  "projectId": 1549,
+  "projectId": 1,
   "clientId": 335,
-  "conversationId": "4c8c7408-f73c-42cd-89e9-afbbee7d9024",
-  "operatorId": 15253,
+  "conversationId": "1",
```
Comment (sdemonti) [inline docs/webhooks.md:56]: must be UUID

File: docs/webhooks.md [hunk 7]
```diff
@@ -199,11 +228,11 @@ Emitted when visitor has answered a customer satisfaction, net promoter score, s
     "eventId": "d36cd3c4-2d16-4a77-97c2-620bde859b29",
     "eventType": "satisfaction.answered",
     "platform": "sd",
-    "projectId": 3030,
+    "projectId": 1,
     "clientId": 1,
     "createdAt": "2017-04-22T11:01:00+02:00",
     "sentAt": "2017-04-22T11:01:00+02:00",
-    "conversationId": "2d90a8a2-16df-45f5-897e-31adf9aa165d",
+    "conversationId": "1",
```
Comment (sdemonti) [inline docs/webhooks.md:235]: Must be UUID

File: docs/webhooks.md [hunk 8]
```diff
@@ -214,11 +243,11 @@ Emitted when visitor has answered a customer satisfaction, net promoter score, s
...
```
Comment (sdemonti) [inline docs/webhooks.md:250]: Must be UUID

File: docs/webhooks.md [hunk 9]
```diff
@@ -246,17 +275,17 @@ Emit when a transaction is attributed to a conversation.
...
+    "projectId": 1,
     "clientId": 1,
     "createdAt": "2017-04-22T11:01:00+02:00",
     "sentAt": "2017-04-22T11:01:00+02:00",
     "transactionId": "1d94c3e8-8cbe-4e46-933d-0433ec339f79",
     "receivedAt": "2017-04-22T11:01:00+02:00",
     "externalId": "transaction-1",
     "amount": 300.20,
-    "conversationId": "2d90a8a2-16df-45f5-897e-31adf9aa165d",
+    "conversationId": "1",
```
Comment (sdemonti) [inline docs/webhooks.md:286]: Must be UUID

File: docs/webhooks.md [hunk 10]
```diff
@@ -158,6 +159,34 @@ Emitted when user is disconnecting to administration or desk.
...
+
+### `user.availability.updated`
+
+Emitted when user's availability is updated.
+
+<pre class="prettyprint lang-js">{
+    "eventId": "d36cd3c4-2d16-4a77-97c2-620bde859b29",
+    "eventType": "user.availability.updated",
+    "platform": "sd",
+    "projectId": 1,
```
Comment (sdemonti) [inline docs/webhooks.md:171]: Availability update has no projectId. We must remove this.

**Review by sdemonti — CHANGES_REQUESTED** [review]

File: docs/webhooks.md [hunk 11]
```diff
@@ -158,6 +159,34 @@ Emitted when user is disconnecting to administration or desk.
...
+    "clientId": 1,
+    "createdAt": "2017-04-22T11:01:00+02:00",
+    "sentAt": "2017-04-22T11:01:00+02:00",
+    "operatorId": 1,
+    "origin": "desk",
+    "availabilities": [{"communicationMode": "CHAT", "status": "AVAILABLE"}]
+
```
Comment (sdemonti) [inline docs/webhooks.md:178]:
```suggestion
```

**Review by sdemonti — APPROVED** [review]

**Pull request merged**